<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LentElem</class>
 <widget class="QWidget" name="LentElem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>388</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="AuthorL">
      <property name="minimumSize">
       <size>
        <width>379</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>379</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Автор</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="TextPostL">
      <property name="minimumSize">
       <size>
        <width>379</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>379</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Текст поста</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="LikeL">
          <property name="minimumSize">
           <size>
            <width>89</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>89</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>89</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>89</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Лайк</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="DislikeL">
          <property name="minimumSize">
           <size>
            <width>89</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>89</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>89</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>89</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Дизлайк</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
